engine/taskmanager/taskq.go

```diff
@@ -89,7 +89,7 @@ func (c *TaskManager) loadTaskQ(ctx context.Context) error {
 	c.options.taskq.queue = q
 	if factoryType == FactoryRedis {
 		if err := q.Consumer().Start(ctx); err != nil {
-			return spverrors.Wrapf(err, "failed to start consuming tasks from redis: %v")
+			return spverrors.Wrapf(err, "failed to start consuming tasks from redis")
```

Comment:
If no arguments are here I think it should be `Wrap` instead of `Wrapf`

Reply:
Good point. Will update in "actual" PR 👍